src/main/java/com/julienvey/trello/impl/TrelloImpl.java

```diff
@@ -479,41 +452,79 @@ public void addUrlAttachmentToCard(String idCard, String url) {
         postForObject(createUrl(ADD_ATTACHMENT_TO_CARD).asString(), new Attachment(url), Attachment.class, idCard);
     }
 
+    @Override
+    public List<Member> addMemberToCard(String idCard, String userId) {
+        List<Member> members = Arrays.asList(postForObject(createUrl(ADD_MEMBER_TO_CARD).asString(),
+                new StringValueHolder(userId), Member[].class, idCard));
```

Comment:
maybe refactor this to avoid using StringValueHolder class? not clear what its value is if it just holds a string.

Reply:
The `StringValueHolder` is just request body representation in Java.
I had a choice between `Map<K, V>` and creating new class. I choose second one, should I refactor it to `Map<K, V>`?

Reply:
what happens if you just provide "userId" as is (String value) without wrapping it?

Reply:
We receiving `400 (Bad Request)` with following response body:

```
Error parsing body
```

There is another way to do that. We can send `userId` in the URL, though query parameter, but in that case we should support `POST` requests with empty request body. What do you think about that ?

Reply:
I see, so ObjectMapper cannot write a pure String value into a json representation since there is no "key" part to it, only value.
and wrapping this into a class most likely uses reflection to detect field name and use it as a key.
I would just use some kind of map for this then, to make this very generic - rather than additional wrappers.

as a side note, I think supporting empty request body is a valid use-case too, although may be outside of the scope of this change.

Reply:
> I would just use some kind of map for this then, to make this very generic - rather than additional wrappers.

I'll use `Map`.

> as a side note, I think supporting empty request body is a valid use-case too, although may be outside of the scope of this change.

Sure.